src/SingleBookingPage/bookingItem/AccordionHeader.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+import * as React from 'react';
+import idx from 'idx';
+import { Typography } from '@kiwicom/orbit-components';
+import {
+  FlightDirect,
+  ChevronDown,
+  ChevronUp,
+} from '@kiwicom/orbit-components/lib/icons';
+import css from 'styled-jsx/css';
+
+import CarrierLogoWrapper from './CarrierLogoWrapper';
+import { FormatDate } from '../../helpers/dateUtils';
+import type { AccordionTripSummary_trip } from './__generated__/AccordionTripSummary_trip.graphql';
+
+const headerStyles = css`
+  .header {
+    display: flex;
+    width: 100%;
+  }
+  span.arrow {
+    margin: 0px 4px;
+  }
+  .logo {
+    margin-right: 12px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+  .toggle {
+    position: relative;
+    margin-left: auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    cursor: pointer;
+  }
+  .itinerary .location {
+    margin-right: 4px;
+  }
+`;
+type Props = {|
+  trip: AccordionTripSummary_trip,
+  toggle: () => void,
```

Comment:
IMHO the whole AccordionHeader should be clickable and this component should accept expanded true/false for chevron icon

Reply:
I agree, whole AccordionHeader would be more comfortable. However let's make sure with @MRTNKCR @mjancik's opinoin :slightly_smiling_face: 

> this component should accept expanded true/false for chevron icon

Do you mean the parent `Accordion` component should hold the `toggled/untoggled` state and pass it as props to `AccordionHeader`? If so, I agree, it makes sense conceptually for `Acoordion` to hold the state.

Reply:
> I agree, whole AccordionHeader would be more comfortable. However let's make sure with @MRTNKCR @mjancik's opinoin

yes, make the whole header clickable

then you don't need to pass down `toggle` callback and `expanded=true/false` should be enough

Reply:
> then you don't need to pass down toggle callback

Then who is the one toggling the `expanded` state?